src/hermes/model/context.py

```diff
@@ -119,6 +120,14 @@ def error(self, ep: EntryPoint, error: Exception):
 
         self._errors.append((ep, error))
 
+    def purge_caches(self) -> Path:
```

Comment:
```suggestion
    def purge_caches(self) -> None:
```

This still doesn't return anything